docs/user/agent/add-node/add-nodes.md

```diff
@@ -100,4 +101,54 @@ extra-worker-0  Ready    worker                  1h   v1.29.3+8628c3c
 master-0        Ready    control-plane,master   31h   v1.29.3+8628c3c
 master-1        Ready    control-plane,master   32h   v1.29.3+8628c3c
 master-2        Ready    control-plane,master   32h   v1.29.3+8628c3c
-```
\ No newline at end of file
+```
+
+# Monitoring
+After the ISO image is booted on a node, progress can be monitored using the node-joiner-monitor.sh script. 
+
+Download the [node-joiner-monitor.sh](./node-joiner-monitor.sh) script to a local directory.
+
+The script requires the IP address of the node to monitor.
+
+Run [node-joiner-monitor.sh](./node-joiner-monitor.sh):
+```bash
+$ ./node-joiner-monitor.sh 192.168.111.90
+```
+
+The script will execute a command to monitor the node using a temporary namespace with
+prefix `openshift-node-joiner-monitor` in the target cluster. The output of this command
+is printed out to stdout.
+
+The monitoring script will print out the CSRs that will need to be approved for the node
```

Comment:
Here it could be useful to cite another couple of relevant points, printed by the script:

* Pre-flight validations. In case the node does not match on or more validations, the installation will not start. The output of the mismatching validations is reported in the stdout to allow the user to fix the problem(s) when required.
* Installation progress. A progress about the current installation stage is reported, it like the image writing to the disk.